modal/cli/run.py

```diff
@@ -692,9 +707,24 @@ def shell(
         modal_image = Image.from_registry(image, add_python=add_python) if image else None
         volumes = {} if volume is None else {f"/mnt/{vol}": Volume.from_name(vol) for vol in volume}
         secrets = [] if secret is None else [Secret.from_name(s) for s in secret]
+
+        mounts = []
+        if add_local:
+            for local_path_str in add_local:
+                local_path = Path(local_path_str).expanduser().resolve()
+                # mount at /{basename}, matching the default of Image.add_local_dir()
```

Comment:
Which default of `.add_local_dir` do you mean? Currently we actually require that you set an (absolute) `remote_path`.

Note that `modal shell --volume` will mount the Volume root at `/mnt/{name}`; not sure if we want to match that.